jOOQ/src/main/java/org/jooq/tools/Convert.java

```diff
@@ -430,7 +430,7 @@ private Convert() {}
     /**
      * The converter to convert them all.
      */
-    private static class ConvertAll<U> implements Converter<Object, U> {
+    static class ConvertAll<U> implements Converter<Object, U> {
 
```

Comment:
I'd prefer not to open this up to package privacy. I understand you did this to access the class from tests, but this class shouldn't be accessible from tests. Tests should only call `Convert.xxx()` methods.